main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"crypto/rsa"
+	"github.com/joho/godotenv"
+	"github.com/ufcg-lsd/arrebol-pb-worker/utils"
+	"github.com/ufcg-lsd/arrebol-pb-worker/worker"
+	"log"
+	"os"
+)
+
+const (
+	ConfFilePathKey = "CONF_FILE"
+	ServerEndpointKey = "SERVER_ENDPOINT"
+)
+
+func setup(serverEndPoint string, workerId string) {
```

Comment:
didn't like this setup function. it is hard to understand what it does by its name. IMHO it would be better to have two functions: `generateKeys` and `sendKeys`